Tests fail to build with VS2015

Run: build.cmd vs2015

C:\coreclr\tests\src\packages\Microsoft.DotNet.BuildTools.1.0.21-prerelease\lib\packageresolve.targets(25,5): error MSB4062: The "ResolveNuGetPackages" task could not be loaded from the assembly C:\coreclr\tests\src\packages\Microsoft.DotNet.BuildTools.1.0.21-prerelease\lib\Microsoft.DotNet.Build.Tasks.dll. Could not load file or assembly 'Microsoft.Build.Utilities.v12.0, Version=12.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a' or one of its dependencies. The system cannot find the file specified. Confirm that the <UsingTask> declaration is correct, that the assembly and all its dependencies are available, and that the task contains a public class that implements Microsoft.Build.Framework.ITask. [C:\coreclr\tests\src\Threading\ThreadStatics\cs_template.proj]

--- a/dir.props
+++ b/dir.props
@@ -9,7 +9,7 @@
   
   <!-- Build Tools Versions -->
   <PropertyGroup>
-    <BuildToolsVersion>1.0.25-prerelease-00056</BuildToolsVersion>
+    <BuildToolsVersion>1.0.25-prerelease-00079</BuildToolsVersion>
     <BuildToolsCoreCLRVersion>1.0.3-prerelease</BuildToolsCoreCLRVersion>
     <DnxVersion Condition="'$(OsEnvironment)'!='Unix'">1.0.0-beta5-12101</DnxVersion>
     <DnxVersion Condition="'$(OsEnvironment)'=='Unix'">1.0.0-beta5-12101</DnxVersion>

--- a/src/.nuget/packages.Unix.config
+++ b/src/.nuget/packages.Unix.config
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <packages>
-  <package id="Microsoft.DotNet.BuildTools" version="1.0.25-prerelease-00056" />
+  <package id="Microsoft.DotNet.BuildTools" version="1.0.25-prerelease-00079" />
   <package id="Microsoft.DotNet.BuildTools.CoreCLR" version="1.0.3-prerelease" />
   <package id="dnx-mono" version="1.0.0-beta5-12101" />
   <package id="Microsoft.Net.ToolsetCompilers" version="1.0.0-rc3-20150510-01" />

--- a/src/.nuget/packages.Windows_NT.config
+++ b/src/.nuget/packages.Windows_NT.config
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <packages>
-  <package id="Microsoft.DotNet.BuildTools" version="1.0.25-prerelease-00056" />
+  <package id="Microsoft.DotNet.BuildTools" version="1.0.25-prerelease-00079" />
   <package id="dnx-coreclr-win-x86" version="1.0.0-beta5-12101" />
   <package id="Microsoft.DotNet.BuildTools.CoreCLR" version="1.0.3-prerelease" />
 </packages>

--- a/tests/dir.props
+++ b/tests/dir.props
@@ -9,7 +9,7 @@
   
   <!-- Build Tools Versions -->
   <PropertyGroup>
-    <BuildToolsVersion>1.0.25-prerelease-00056</BuildToolsVersion>
+    <BuildToolsVersion>1.0.25-prerelease-00079</BuildToolsVersion>
     <BuildToolsCoreCLRVersion>1.0.2-prerelease</BuildToolsCoreCLRVersion>
     <DnxVersion Condition="'$(OsEnvironment)'!='Unix'">1.0.0-beta5-12101</DnxVersion>
     <DnxVersion Condition="'$(OsEnvironment)'=='Unix'">1.0.0-beta5-12101</DnxVersion>

--- a/tests/publishdependency.targets
+++ b/tests/publishdependency.targets
@@ -1,14 +1,14 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="12.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
 
-  <UsingTask TaskName="ResolveNuGetPackageAssets" AssemblyFile="$(ToolsDir)Microsoft.DotNet.Build.Tasks.dll"/>
-  
+  <UsingTask TaskName="PrereleaseResolveNuGetPackageAssets" AssemblyFile="$(ToolsDir)Microsoft.DotNet.Build.Tasks.dll"/>
+
   <ItemGroup>
     <TestTargetFramework Include="DNXCore,Version=v5.0">
       <Folder>dnxcore50</Folder>
     </TestTargetFramework>
   </ItemGroup>
-  
+
   <ItemGroup>
     <AllProjectJsonLockFiles Include="$(SourceDir)**\*\project.lock.json"/>
   </ItemGroup>
@@ -28,20 +28,19 @@
       <ProjectLockJsonFiles Include="@(AllProjectJsonLockFiles)"  Exclude="@(ProjectJsonLockFileExclude)"/>
     </ItemGroup>
     
-    <!-- This will use the overridden ResolveNuGetPackageAssets, which outputs copy local items
+    <!-- This will use the overridden PrereleaseResolveNuGetPackageAssets, which outputs copy local items
          for the xunit wrapper projects -->
-    <ResolveNuGetPackageAssets  Architecture="$(PlatformTarget)"
-                                Configuration="$(Configuration)"
-                                Language="$(Language)"
-                                PackageRoot="$(PackagesDir)"
-                                ProjectLockFile="%(ProjectLockJsonFiles.Identity)"
-                                TargetFrameworkMonikers="@(TestTargetFramework)"
-                                TargetPlatformMonikers="$(TargetPlatformMoniker)">
-
+    <PrereleaseResolveNuGetPackageAssets AllowFallbackOnTargetSelection="true"
+                                         IncludeFrameworkReferences="false"
+                                         NuGetPackagesDirectory="$(PackagesDir)"
+                                         RuntimeIdentifier="dotnet"
+                                         ProjectLanguage="$(Language)"
+                                         ProjectLockFile="%(ProjectLockJsonFiles.Identity)"
+                                         TargetMonikers="@(TestTargetFramework)">
       <Output TaskParameter="ResolvedAnalyzers" ItemName="Analyzer" />
       <Output TaskParameter="ResolvedReferences" ItemName="Reference" />
       <Output TaskParameter="ResolvedCopyLocalItems" ItemName="RunTimeCopyLocal" />
-    </ResolveNuGetPackageAssets>
+    </PrereleaseResolveNuGetPackageAssets>
 
     <ItemGroup>
       <RunTimeDependecyExclude Include="$(CORE_ROOT)\**\*.*"  />

--- a/tests/src/.nuget/packages.Windows_NT.config
+++ b/tests/src/.nuget/packages.Windows_NT.config
@@ -1,5 +1,5 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
-  <package id="Microsoft.DotNet.BuildTools" version="1.0.25-prerelease-00056" />
+  <package id="Microsoft.DotNet.BuildTools" version="1.0.25-prerelease-00079" />
   <package id="dnx-coreclr-win-x86" version="1.0.0-beta5-12101" />
 </packages>

--- a/tests/xunitwrapper.targets
+++ b/tests/xunitwrapper.targets
@@ -4,20 +4,20 @@
      to the core_root directory -->
 
 <Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-  <Target Name="ResolveNuGetPackages" Condition="'$(ResolveNuGetPackages)'=='true'">
-    
-    <ResolveNuGetPackageAssets Condition="Exists($(ProjectLockJson))"
-                               Architecture="$(PlatformTarget)"
-                               Configuration="$(Configuration)"
-                               Language="$(Language)"
-                               PackageRoot="$(PackagesDir)"
-                               ProjectLockFile="$(ProjectLockJson)"
-                               TargetFrameworkMonikers="$(NugetTargetFrameworkMoniker)"
-                               TargetPlatformMonikers="$(TargetPlatformMoniker)">
+  <Target Name="ResolveNuGetPackages">
+
+    <PrereleaseResolveNuGetPackageAssets Condition="Exists($(ProjectLockJson))"
+                                         AllowFallbackOnTargetSelection="true"
+                                         IncludeFrameworkReferences="false"
+                                         NuGetPackagesDirectory="$(PackagesDir)"
+                                         RuntimeIdentifier="dotnet"
+                                         ProjectLanguage="$(Language)"
+                                         ProjectLockFile="$(ProjectLockJson)"
+                                         TargetMonikers="$(NugetTargetFrameworkMoniker)">
       <Output TaskParameter="ResolvedAnalyzers" ItemName="Analyzer" />
       <Output TaskParameter="ResolvedReferences" ItemName="Reference" />
       <Output TaskParameter="ResolvedCopyLocalItems" ItemName="CopyLocal" />
-    </ResolveNuGetPackageAssets>
+    </PrereleaseResolveNuGetPackageAssets>
 
     <!-- We may have an indirect package reference that we want to replace with a project reference -->
     <ItemGroup>

